website/docs/blocks.md

```diff
@@ -2,7 +2,7 @@
 
 Devnet starts with a genesis block (with a block number equal to 0). In forking mode, the genesis block number will be equal to the forked block number plus one.
 
-A new block is generated based on the pending block, once a new block is generated the pending block is restarted. By default, a new block is generated with each new transaction, but you can also [create an empty block by yourself](#create-an-empty-block).
```

Comment:
why this was removed "A new block is generated based on the pending block, once a new block is generated the pending block is restarted." wrong resolved conflict?

Reply:
I see you resolved, but I felt the need to improve blocks.md: https://github.com/0xSpaceShard/starknet-devnet-rs/pull/515/commits/1873c579df9e1f520582dc788d67911c556c91be